awx/api/filters.py

```diff
@@ -292,39 +278,6 @@ def filter_queryset(self, request, queryset, view):
                     key = key[5:]
                     q_not = True
 
-                # Make legacy v1 Job/Template fields work for backwards compatability
-                # TODO: remove after API v1 deprecation period
-                if queryset.model._meta.object_name in ('JobTemplate', 'Job') and key in (
-                        'credential', 'vault_credential', 'cloud_credential', 'network_credential'
```

Comment:
I had to check that the UI TEMPLATES view isn't showing any of these keys, and it seems that it does not, so good there.